generators_and_lazy_evaluation.ipynb

```diff
@@ -0,0 +1,667 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Lets count the actual calls to
</pre><p>Typo:<strong> Let's </strong> count the actual calls <strong>too</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuefische/ds-meetups/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>